frontend/src/pages/DegreeWizard/YearStep/YearStep.tsx

```diff
@@ -19,6 +18,11 @@ type Props = {
   setDegreeInfo: SetState<DegreeWizardPayload>;
 };
 
+// Disable 10 years from the selected date
+const disable10YearsOnwards: DatePickerProps['disabledDate'] = (current, { from }) => {
+  return from ? Math.abs(current.year() - from.year()) >= 10 : false;
```

Comment:
```suggestion
  return from !== undefined && Math.abs(current.year() - from.year()) >= 10;
```
This is a bit cleaner, as we wer saying "if from exists, then its greater than 10 years, else false" which is equivalent to "if from exists and year is greater than 10 years".